gapis/api/vulkan/api/enums.api

```diff
@@ -375,9 +375,15 @@ enum VkStructureType: u32 {
   VK_STRUCTURE_TYPE_ATTACHMENT_REFERENCE_STENCIL_LAYOUT_KHR                     = 1000241001,
   VK_STRUCTURE_TYPE_ATTACHMENT_DESCRIPTION_STENCIL_LAYOUT_KHR                   = 1000241002,
 
+  // @extension("VK_KHR_depth_stencil_resolve")
+  VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DEPTH_STENCIL_RESOLVE_PROPERTIES_KHR = 1000199000,
```

Comment:
We don't need to repeat this as this is a core extension. We shall only use the core version, as the values are same.

Reply:
Done.